.travis.yml

```diff
@@ -88,7 +88,7 @@ jobs:
         - bundle exec rake spec
       before_deploy:
         - "export PUPMOD_METADATA_VERSION=`ruby -r json -e \"puts JSON.parse(File.read('metadata.json')).fetch('version')\"`"
-        - '[[ $TRAVIS_TAG =~ ^simp-${PUPMOD_METADATA_VERSION}$|^${PUPMOD_METADATA_VERSION}$ ]]'
+        - '[[ "${TRAVIS_TAG}" =~ ^simp-${PUPMOD_METADATA_VERSION}(-.+)?$|^${PUPMOD_METADATA_VERSION}(-.+)?$ ]]'
```

Comment:
Why allow 'simp-x.y.z-r'? Our latest tagging scheme is 'x.y.z-r'.

Reply:
Regex doesn't work as expected, if the intent is to force a release qualifier.  In other words, because '?' is 0 or 1 time, TRAVIS_TAG='6.3.3' would work.